draft-gpew-priv-ppm.md

```diff
@@ -713,25 +719,23 @@ the first aggregate request. Shall we exclude it in subsequent requests
 somehow?] The remainder of the structure is dedicated to VDAF-specific request
 parameters.
 
-In order to provide replay protection, the leader is required to send aggregate
-sub-requests in ascending order, where the ordering on sub-requests is
-determined by the algorithm defined in {{anti-replay}}. Specifically, the leader
-constructs its request such that:
-
-* each sub-request follows the previous sub-request; and
-* the first sub-request follows the last sub-request in the previous aggregate
-  request.
+In order to provide replay protection, the leader preprocesses the set of
+reports it sends in the the AggregateReq as described in {{anti-replay}}. Any
+reports filtered out by this procedure MUST be ignored.
 
 The helper handles well-formed requests as follows. (As usual, malformed
 requests are handled as described in {{errors}}.) It first looks for PPM
-parameters corresponding to `AggregateReq.task_id`. It then filters out
-out-of-order sub-requests by ignoring any sub-request that does not follow the
-previous one (See {{anti-replay}}.)
+parameters corresponding to `AggregateReq.task_id`. It then preprocesses the
+sub-requests as described in {{anti-replay}}. Any sub-requests filtered out by
+this procedure MUST be ignored.
+
+In addition, the helper MUST ignore any report whose nonce contains a timestamp
+that falls in a batch interval for which it has received at least one
+output-share request from the leader. (See {{output-share-request}}.)
```

Comment:
```suggestion
output-share request from the leader. (See {{output-share-request}}.)
This means leaders cannot interleave a sequence of aggregate and 
output-share requests for a single batch.
```

We should probably add guidance for the leader here as to _when_ they should issue their output-share request. If they do it as soon as the minimum batch count is hit, then nothing else in the batch can be aggregated. (We can do this in a followup PR, I think.)